packages/playwright-core/src/utils/index.ts

```diff
@@ -43,3 +43,4 @@ export * from './userAgent';
 export * from './zipFile';
 export * from './zones';
 export * from './isomorphic/locatorGenerators';
+export { escapeRegExp } from './isomorphic/stringUtils';
```

Comment:
Need to explicitly export just this function as otherwise isString clashes with the one exported from ./rtti.